core/app/modules/site-editor/module.php

```diff
@@ -31,13 +31,13 @@ public function add_menu_in_admin_bar( $admin_bar_config ) {
 			'title' => __( 'Theme Builder', 'elementor' ),
 			'sub_title' => __( 'Site', 'elementor' ),
 			'href' => Plugin::$instance->app->get_settings( 'menu_url' ),
-			'class' => 'elementor-app-link',
+			'class' => 'elementor-general-section elementor-app-link',
 		];
 
 		return $admin_bar_config;
 	}
 
 	public function __construct() {
-		add_filter( 'elementor/frontend/admin_bar/settings', [ $this, 'add_menu_in_admin_bar' ] );
+		add_filter( 'elementor/frontend/admin_bar/settings', [ $this, 'add_menu_in_admin_bar' ], 0 /* After kit (Site settings) */ );
```

Comment:
Remove. The default is 10, so it will be after the kit.

Reply:
Handled.